### ISSUE TITLE
IOS Crash "Could not find a storyboard named ..."

### ISSUE THREAD

**Issue by osov** [defold/defold#9226]
**Describe the bug (REQUIRED)**
After assembling and launching on the iPhone, the application crashes.
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Could not find a storyboard named 'LaunchScreenBlack' in bundle NSBundle </private/var/containers/Bundle/Application/2B2B0797-1EE0-46C9-A1C4-B3B0A2E3C99F/Шашки.app> (loaded)'
```
I also noticed that the default link to the screen with the extension .storyboard**с** is now used

**To Reproduce (REQUIRED)**
the project had its own StoryBoard set launch_screen = /project_settings/LaunchScreenBlack.storyboard
Before this, this was not the case in previous versions of Defold.

**Defold version (REQUIRED):**
  - Version 1.9.1

**Platforms (REQUIRED):**
 - Platforms:  iOS
 - OS: iOS17.5.1
 - Device: iPhone14

[LaunchScreenBlack.zip](https://github.com/user-attachments/files/16373901/LaunchScreenBlack.zip)

**Comment by AGulev** [comment]
it seems like new xcode has new storyboard format

**Comment by AGulev** [comment]
@osov is it still an issue for you?

**Comment by osov** [comment]
No, I don't use custom screens anymore and there is no problem.
